### PR TITLE
Add ability to refresh select fields for openstack cached entities

### PR DESCRIPTION
**Pull request by jcwest**
Add ability to refresh select fields for openstack cached entities.
Refactor select fields to consolidate common functionality.
Add select field directives for load balancers and instance types.

### CODE REVIEW

File: app/scripts/modules/openstack/instance/osInstanceTypeSelectField.directive.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.openstack.instance', [
```
Comment (anotherchrisberry) [inline app/scripts/modules/openstack/instance/osInstanceTypeSelectField.directive.js]: Might want to make the package more specific to the component, e.g. `spinnaker.openstack.instance.instanceTypeSelectField`

File: app/scripts/modules/openstack/common/refresh.tooltip.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<p ng-if="$ctrl.refreshing">{{$ctrl.label}} list is <strong>refreshing</strong>.</p>
+<p ng-if="!$ctrl.refreshing">Click <span class="small glyphicon glyphicon-refresh"></span> to refresh {{$ctrl.label}} list.</p>
+<p>Last Updated:<br>{{$ctrl.lastRefresh | timestamp }}<br>({{$ctrl.lastRefresh | relativeTime }})</p>
+<p class='small'><strong>Note:</strong> Due to caching, data may be delayed up to 2 minutes</p>
```
Comment (anotherchrisberry) [inline app/scripts/modules/openstack/common/refresh.tooltip.html]: add newline at the end of the file

**Comment by anotherchrisberry** [comment]
couple of very small things but LGTM otherwise

**Comment by jcwest** [comment]
Addressed the 2 items. Should be good to merge now.

**Pull request merged**